fluentd/configs.d/filter-post-z-retag-two.conf

```diff
@@ -4,12 +4,16 @@
   @label @OUTPUT
 </match>
 
-<match journal.** system.var.log** **_default_** **_openshift_** **_openshift-infra_**>
+# Remove audit logs where type != virt-control
+@include filter-exclude-audit.conf
```

Comment:
This should happen much earlier in the processing - e.g. as `filter-pre-exclude-audit.conf` so that we skip all other processing